alfresco-solr-query-analytics/src/main/java/eu/xenit/alfresco/instrumentation/solr/DebugSolrQueryHTTPClientAdvisor.java

```diff
@@ -0,0 +1,110 @@
+package eu.xenit.alfresco.instrumentation.solr;
+
+import eu.xenit.util.SolrQueryParser;
+import eu.xenit.util.SolrTimingParser;
+import java.lang.reflect.Method;
+import java.util.Map;
+import java.util.Properties;
+import org.alfresco.repo.search.impl.lucene.JSONResult;
+import org.alfresco.repo.search.impl.lucene.SolrJSONResultSet;
+import org.alfresco.service.cmr.search.ResultSet;
+import org.alfresco.service.cmr.search.SearchParameters;
+import org.aopalliance.intercept.MethodInterceptor;
+import org.aopalliance.intercept.MethodInvocation;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.aop.framework.Advised;
+import org.springframework.aop.support.DefaultPointcutAdvisor;
+import org.springframework.util.ReflectionUtils;
+
+public class DebugSolrQueryHTTPClientAdvisor {
+
+    private static final Logger logger = LoggerFactory.getLogger(DebugSolrQueryHTTPClientAdvisor.class);
+
+    private final Properties globalProperties;
+    private final SolrQueryParser solrQueryParser;
+
+    public DebugSolrQueryHTTPClientAdvisor(final Advised bean, Properties globalProperties,
```

Comment:
since it is "injected" via an xml file [https://github.com/xenit-eu/alfresco-solr-query-analytics/blob/9b48672a28809ac465ecaed9214724899f95c755/alfresco-solr-query-analytics/src/main/amp/config/alfresco/module/alfresco-solr-query-analytics/module-context.xml#L12](url) than it should be ok without qualifiers, otherwise the "advised" subsystem proxy would also need the @qualifier